### PR TITLE
Add rubygem-redfish_client package

### PR DESCRIPTION
**Pull request by lzap**
This needs to be in proxy repos for Redfish BMC integration.

* Nightly

Thanks!

### CODE REVIEW

**Comment by lzap** [comment]
I guess I need to wait until the dependency merged few hours ago: https://github.com/theforeman/foreman-packaging/pull/5720

gets generated/published?

```
[2020-09-24T11:30:44.466Z]     Added downloaded_rpms repo from ./downloaded_rpms/rhel7
[2020-09-24T11:30:44.466Z]     package: tfm-rubygem-redfish_client-0.5.2-1.el7.noarch from downloaded_rpms
[2020-09-24T11:30:44.466Z]       unresolved deps:
[2020-09-24T11:30:44.466Z]         tfm-rubygem(server_sent_events) < 1
[2020-09-24T11:30:44.466Z]         tfm-rubygem(server_sent_events) >= 0.1
```

**Comment by evgeni** [comment]
[test rpm]

**Review by evgeni — APPROVED** [review]

**Comment by lzap** [comment]
Thanks!

**Pull request merged**
